app/views/beta/search_results/_sidebar.html.erb

```diff
@@ -34,5 +34,7 @@
       </div>
 
     <% end %>
+
+    <%= render "search/guide_section", guide: guide if guide %>
```

Comment:
I guess it's better to use
`... if guide.present?` 
to cover also few more cases that are considered empty (like empty string)